docker/entrypoint.sh

```diff
@@ -73,15 +107,34 @@ while true; do
     shift
 done
 
-echo "$APP_ARGS" >/etc/weave/scope-app.args
-echo "$PROBE_ARGS" >/etc/weave/scope-probe.args
+if is_running $WEAVE_CONTAINER_NAME; then
+    container_ip $WEAVE_CONTAINER_NAME
+    PROBE_ARGS="$PROBE_ARGS -weave.router.addr=$CONTAINER_IP"
+    weave_expose
 
-if [ -n "$DNS_SERVER" -a -n "$SEARCHPATH" ]; then
-    echo "domain $SEARCHPATH" >/etc/resolv.conf
-    echo "search $SEARCHPATH" >>/etc/resolv.conf
-    echo "nameserver $DNS_SERVER" >>/etc/resolv.conf
+    DOCKER_BRIDGE_IP=$(docker_bridge_ip)
+    echo "Weave container detected at $CONTAINER_IP, Docker bridge at $DOCKER_BRIDGE_IP"
+
+    echo "domain $DOMAIN" >/etc/resolv.conf
+    echo "search $DOMAIN" >>/etc/resolv.conf
+    echo "nameserver $DOCKER_BRIDGE_IP" >>/etc/resolv.conf
```

Comment:
Won't this have been set up when the scope probe was launched into the weave network? Or are we not assuming that it is in the weave network?